registry/storage/driver/middleware/cloudfront/s3filter_test.go

```diff
@@ -62,7 +62,7 @@ func TestS3TryUpdate(t *testing.T) {
 	})
 	defer server.Close()
 
-	ips := newAWSIPs(serverIPRanges(server), time.Hour, nil)
+	ips, _ := newAWSIPs(context.Background(), serverIPRanges(server), time.Hour, nil)
```

Comment:
question: I presume the reason why we're ignoring errors here is because `serverIPRanges` returns a `string` and doesnt actually make any network requests?

Reply:
The error being ignored is from `newAWSIPs`, which _does_ make network requests. Though it does not feel the need to return network errors for some reason. I changed the signature of `newAWSIPs` to return an error to follow my rule that cancelation must be detectable by the caller, and got lazy updating the tests to assert for the absence of an error which cannot be triggered with the current tests and implementation.